### ISSUE TITLE
Annotation feature confusion

### ISSUE THREAD

**Issue by computron** [materialsintelligence/matstract#63]
For the annotation feature, I am not sure what the "Inorganic Crystal" tag refers to. Does this mean "crystal structure" information (like "rutile")? Or symmetry information (like "spacegroup 145")? Or something else entirely?

**Comment by jdagdelen** [comment]
"Inorganic crystal" in this case means the name/formula of an inorganic crystalline material (for example "TiO2" or "graphene". We trying to use this annotation to improve our ability to extract materials that are likely to be found in the materials project database.

**Comment by computron** [comment]
I see - so an "Inorganic crystal" is also a "material" and potentially also a "main material"?

I think my preference (feel free to do it your own way if you disagree) would be to separate out some of the annotations, otherwise it is a lot of mental load to annotate even a simple abstract. For example, you could:

 - remove "Inorganic crystal" and "main material" from the main annotations - the user doesn't worry about this distinction for now
- a separate annotation tool can show you a list of materials (maybe 20 materials at once, potentially compiled from many different abstracts) and ask you to click which ones also represent inorganic crystals

Thus, the mental task of figuring out what are materials within a text is separated from the mental task of distinguishing organic / inorganic / crystalline materials.

Similarly, for the "main material", it can be a separate tool that shows you an abstract with all detected materials highlighted and what your algorithm thinks is the main material highlighted in a different color. It then just asks you a simple question: "is XXXX the main material (or one of several equally primary materials) in the study?" to verify if the proposed classification was correct or not. You just click Yes/No. If you click No you can have the user click the main material or just move on the next abstract (in the latter case, the user can just quickly answer yes/no and not have to worry about doing anything else)

**Comment by computron** [comment]
Basically, speaking for myself, I can say that I can do a task like annotation pretty quickly and rapidly across 100s of articles if it's always the same mental task. I can get into a mode where I just focus in on that one task and quickly do yes/no/yes/no etc 100s of times. But if each annotation is multiple separate tasks that involves different modes of thoughts and regions of my brain, then I get really stuck and mired in a single abstract and need to double-check everything, can potentially forget to do a step, etc. and it might take me 5 minutes to do a single abstract which is no fun.

**Comment by computron** [comment]
Note also for this comment:

```
a separate annotation tool can show you a list of materials (maybe 20 materials at once, potentially compiled from many different abstracts) and ask you to click which ones also represent inorganic crystals
```

You can also have the system guess inorganic/organic to prefill the system for the user, and the user can just click on the ones that need to be changed or just click verify if all 20 are correct.

**Comment by jdagdelen** [comment]
I agree with Anubhav on this. When I tried to do some annotations, I had a similar experience of mental friction. Let's discuss this in the next Tuesday meeting.

**Comment by vtshitoyan** [comment]
I have created a document trying to summarize most of the points made by @computron and others. You can find it [here](https://docs.google.com/document/d/1Riaj23L4_hCknXxuyZb9Rnp74bSoaGfHy0hZC5e4Wzc/edit?usp=sharing). Some definitions might still have to be clarified, but seems like we can distribute everything across 6 different annotation tasks (see task grouping at the end of the document).

**Comment by computron** [comment]
Looks great! My minor comments:

- add a rough wiki or other document to set standards for things like relevant / not relevant, i.e. so annotations are consistent across people
- have "not relevant" be an option for all tagging tasks rather than just the main one (or keep it the same as you have now if you like it the current way, this is not a strong opinion at all)
- have "this abstract looks weird" or something as an option for all tagging tasks, i.e., there are some abstracts that look incomplete or chopped off when I tried it

**Comment by computron** [comment]
Actually on second thought ignore my second bullet about “not relevant”, its probably better the way it is now (relevance up front)
